### ISSUE TITLE
Training for compcars

### ISSUE THREAD

**Issue by huangqiusheng** [facebookresearch/StyleNeRF#15]
Hi, many thanks to the author for actively releasing the pre-trained model. I need a car model recently, so I processed the comcars dataset as suggested by the author, and trained for three days under 4 v100 (32GB), but the result is very poor, what should I do.
My training code is “python run_train.py outdir=/StyleNeRF-main/exp_car data=/CompCars/data/img_process_train spec=paper256 model=stylenerf_cars”. 
[The result video](https://drive.google.com/file/d/1rzpk86PvFTUifwae43RH8IBUM7dy4pPa/view?usp=sharing)
Also, we still expect the author to release the pre-trained model of CompCars as soon as possible. Thank you for your help!

**Comment by huangqiusheng** [comment]
By the way, when I train the network for the first time, there is a mode collapse after two days.

**Comment by MultiPath** [comment]
Yes, training of compcars is very unstable. I have a checkpoint which is not uploaded yet. Even for that checkpoint, many examples failed.

**Comment by huangqiusheng** [comment]
In fact, we tried a lot of samples and couldn't find even one that came close to the author's video. So we can only hope that you release the checkpoint as soon as possible. Thanks a lot.

**Comment by ZHDphilip** [comment]
How did you preprocess the dataset? Did you put all images in one large folder then run dataset_tool.py as specified in the stylegan2 ada repo?

**Comment by MultiPath** [comment]
Yes, just follow the stylegan2 ada repo did.

**Comment by ZHDphilip** [comment]
How did you deal with the label? I am also trying to train on compcars, and I'm having trouble dealing with the labels

**Comment by MultiPath** [comment]
We did not use labels in the current version.

**Comment by ZHDphilip** [comment]
Thanks for the clarification. Will follow up with training results

**Comment by techie02** [comment]
@huangqiusheng 
I am also trying to train the model for compcars. I am able to reach FID score of around 68. After around 3600kimg FID starts to increase. Could you let me know what was the FID score that you were able to reach and for how many iterations did you train it.
Thanks

**Comment by Jain-Archit** [comment]
@huangqiusheng @techie02 does anyone have a stable model for compcars available for public use?
